.github/pull_request_template.md

```diff
@@ -4,6 +4,10 @@ PR Title Format:
 1. pkg [, pkg2, pkg3]: what's changed
 2. *: what's changed
 
+If you want to open the **Challenge Program** pull request, please use the following template:
+https://raw.githubusercontent.com/pingcap/.github/master/.github/PULL_REQUEST_TEMPLATE/challenge-program.md
```

Comment:
what's the difference between these two PR templates?

Reply:
> could you prepare a different template for each type of issue? It's better to use full names instead of just abbr to eliminate confusion for new contributors.

I think abbreviations can be used, and I will add a document link to the template(pingcap/.github).

> what's the difference between these two PR templates?

For now, it is the same, but we may adjust the PR template of the challenge program later, so we recommend using a global organization template.